### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Illia-L**
[DEMO LINK](https://illia-l.github.io/layout_landing-page/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,350 @@
...
+              </a>
+            </div>
+
+            <h3 class="product-group__title">
+              Smart Home
+            </h3>
+          </article>
+
+          <a class="section-catalog-preview__link-view-all" href="#">
+            <button class="btn">
```
Comment (anastasiiavorobiova) [inline src/index.html:240]: The element button must not appear as a descendant of an "a" element accordingly to the living standard.

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Really nice looking! 
Screenshot 2023-06-18 at 21 06 59
1) Consider adding a white background to the top of the section. 2) Consider making links follow the anchor sections
Screenshot 2023-06-18 at 21 08 00
3) Fix indentations to match the Figma design

**Review by polosanya — CHANGES_REQUESTED** [review]
Keep up the good work!
1. There should be no white stripes. Content should take the whole possible width.
Also, the horizontal scroll is present for some reason. Please fix it
image
2. Form doesn't match the design
image
image
3. Make sure that all interactive elements have cursor pointer on hover
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Need to check font-family here from design
image

There is no option to call on click
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Do you need this white border? Remove it
image

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Hello!
Nice work!

### To improve:
use svg for all of your icons instead of png

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Almost done! Fix the menu, please 

- Seems that the menu items are different from the design. Fix it, please  
image
image

- Use SVG for the logo to improve the quality 
image

**Review by volodymyr-soltys97 — APPROVED** [review]
Nice work 🔥

**Pull request closed without merging**